site/src/pages/DeploySettingsPage/NotificationsPage/NotificationsPage.tsx

```diff
@@ -0,0 +1,285 @@
+  });
+  const ready =
+    templatesByGroup.data && dispatchMethods.data && deploymentValues;
+  const tab = searchParams.get("tab") || "events";
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("Notifications Settings")}</title>
+      </Helmet>
+      <Section
+        title="Notifications"
+        description={
+          <>
+            Control delivery methods for notifications on this deployment.
+            Notifications may be disabled in your{" "}
+            <Link
+              to="/settings/notifications"
+              css={(theme) => ({
+                color: theme.roles.active.fill.outline,
+                textDecoration: "none",
+                "&: hover": {
+                  textDecoration: "underline",
+                },
+              })}
+            >
+              profile settings
+            </Link>
+            .
+          </>
+        }
+        layout="fluid"
+      >
+        <Tabs active={tab}>
+          <TabsList>
+            <TabLink to="?tab=events" value="events">
+              Events
+            </TabLink>
+            <TabLink to="?tab=settings" value="settings">
+              Settings
+            </TabLink>
+          </TabsList>
+        </Tabs>
+
+        <div css={styles.content}>
+          {ready ? (
+            tab === "events" ? (
+              <EventsView
+                templatesByGroup={templatesByGroup.data}
+                deploymentValues={deploymentValues.config}
+                defaultMethod={castNotificationMethod(
+                  dispatchMethods.data.default,
+                )}
+                availableMethods={dispatchMethods.data.available.map(
+                  castNotificationMethod,
+                )}
+              />
+            ) : (
+              <OptionsTable
+                options={deploymentValues?.options.filter((o) =>
+                  deploymentGroupHasParent(o.group, "Notifications"),
+                )}
+              />
+            )
+          ) : (
+            <Loader />
+          )}
+        </div>
+      </Section>
+    </>
+  );
+};
+
+type EventsViewProps = {
+  defaultMethod: NotificationMethod;
+  availableMethods: NotificationMethod[];
+  templatesByGroup: ReturnType<typeof selectTemplatesByGroup>;
+  deploymentValues: DeploymentValues;
+};
+
+const EventsView: FC<EventsViewProps> = ({
+  defaultMethod,
+  availableMethods,
+  templatesByGroup,
+  deploymentValues,
+}) => {
+  return (
+    <Stack spacing={4}>
+      {availableMethods.includes("smtp") &&
+        deploymentValues.notifications?.webhook.endpoint === "" && (
+          <Alert severity="warning">
+            Webhook notifications are enabled, but no endpoint has been
+            configured.
+          </Alert>
+        )}
+
+      {availableMethods.includes("smtp") &&
+        deploymentValues.notifications?.email.smarthost === "" && (
+          <Alert severity="warning">
+            SMTP notifications are enabled, but no smart host has been
```

Comment:
> I think we should link to the documentation here, or at least mention the config option to use.

I totally agree. Do we already have a link for the docs?

Reply:
@dannykopping, it looks like the docs aren't ready yet, so I'm going to merge the PR and add the link to the docs later. Does that sound good? I updated the description of https://github.com/coder/team-coconut/issues/19 to reflect this decision.

Reply:
I'm on mobile otherwise I'd link it, but have a look on the docs site. We just need to include the env variable name and a link to it on the docs site